### PR TITLE
[Analysis services] Polish

### PR DESCRIPTION
**Pull request by mcgallan**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: .vscode/cspell.json [hunk 1]
```diff
@@ -106,6 +106,13 @@
         "warnaserror"
       ]
     },
+    {
+      "filename": "**/sdk/analysisservices/**/*.cs",
+      "words": [
+        "Dmts",
+        "MSAL"
```
Comment (fengzhou-msft) [inline .vscode/cspell.json:113]: I didn't find `MSAL` in `*.cs` files. Do we still need it?

File: sdk/analysisservices/Azure.ResourceManager.AnalysisServices/api/Azure.ResourceManager.AnalysisServices.netstandard2.0.cs [hunk 2]
```diff
@@ -122,15 +166,21 @@ public partial class CheckServerNameAvailabilityContent
...
     }
     public enum ConnectionMode
     {
         All = 0,
         ReadOnly = 1,
     }
+    public partial class ExistingResourceSkuDetails
+    {
+        internal ExistingResourceSkuDetails() { }
+        public string ResourceType { get { throw null; } }
```
Comment (fengzhou-msft) [inline sdk/analysisservices/Azure.ResourceManager.AnalysisServices/api/Azure.ResourceManager.AnalysisServices.netstandard2.0.cs:181]: Can this property use `ResourceType` type?

File: sdk/analysisservices/Azure.ResourceManager.AnalysisServices/api/Azure.ResourceManager.AnalysisServices.netstandard2.0.cs [hunk 3]
```diff
@@ -153,8 +203,8 @@ public partial class IPv4FirewallRule
     public partial class IPv4FirewallSettings
     {
         public IPv4FirewallSettings() { }
-        public bool? EnablePowerBIService { get { throw null; } set { } }
         public System.Collections.Generic.IList<Azure.ResourceManager.AnalysisServices.Models.IPv4FirewallRule> FirewallRules { get { throw null; } }
+        public bool? IsEnablePowerBIService { get { throw null; } set { } }
```
Comment (fengzhou-msft) [inline sdk/analysisservices/Azure.ResourceManager.AnalysisServices/api/Azure.ResourceManager.AnalysisServices.netstandard2.0.cs:207]: Please use `IsPowerBIServiceEnbaled`.

File: .vscode/cspell.json (hunk 1 above)
Comment (mcgallan) [reply .vscode/cspell.json:113]: fixed

File: sdk/analysisservices/Azure.ResourceManager.AnalysisServices/api/Azure.ResourceManager.AnalysisServices.netstandard2.0.cs (hunk 3 above)
Comment (mcgallan) [reply sdk/analysisservices/Azure.ResourceManager.AnalysisServices/api/Azure.ResourceManager.AnalysisServices.netstandard2.0.cs:207]: fixed

File: sdk/analysisservices/Azure.ResourceManager.AnalysisServices/api/Azure.ResourceManager.AnalysisServices.netstandard2.0.cs (hunk 2 above)
Comment (mcgallan) [reply sdk/analysisservices/Azure.ResourceManager.AnalysisServices/api/Azure.ResourceManager.AnalysisServices.netstandard2.0.cs:181]: fixed

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**